Content.Server/_NF/Clothing/EntitySystems/AutoSuitSensorOffSystem.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Server._NF.Medical.SuitSensors;
+using Content.Server.Medical.SuitSensors;
+using Content.Shared.Access.Components;
+using Content.Shared.Clothing._NF.Components;
+using Content.Shared.Clothing.Components;
+using Content.Shared.Clothing.EntitySystems;
+using Content.Shared.Inventory;
+using Content.Shared.Medical.SuitSensor;
+using Content.Shared.Roles;
+
+namespace Content.Server._NF.Clothing.EntitySystems;
```

Comment:
```suggestion
namespace Content.Server._NF.Medical.SuitSensors;
```
Keep the namespace the same.

Also move the file :pray: